src/StateProvider.js

```diff
@@ -121,13 +121,19 @@ export default class StateProvider {
 
       static displayName = `ProvideState(${getDisplayName(Component)})`;
 
+      isSubscribed() {
+        return typeof this.unsubscribe === 'function';
```

Comment:
Took it from here and thought it was nice: https://github.com/reactjs/react-redux/commit/bda582b728514446594bcef3c2f39ce52dba02d9

I can easily inline it :)